meson.build

```
@@ -0,0 +1,28 @@
+project('diffuse',
+    version: '0.6.0',
+    meson_version: '>= 0.50',
+    license: 'GPL2+')
+
+i18n = import('i18n')
+python = import('python')
+
+py_installation = python.find_installation('python3')
+
+find_program('gtk-update-icon-cache', required: false)
+
+prefix = get_option('prefix')
+
+bindir = join_paths(prefix, get_option('bindir'))
+datadir = join_paths(prefix, get_option('datadir'))
+localedir = join_paths(prefix, get_option('localedir'))
+libexecdir = join_paths(prefix, get_option('libexecdir'))
+sysconfdir = join_paths(prefix, get_option('sysconfdir'))
```

```suggestion
bindir = join_paths(prefix, get_option('bindir'))
datadir = prefix / get_option('datadir')
localedir = prefix / get_option('localedir')
libexecdir = prefix / get_option('libexecdir')
sysconfdir =prefix / get_option('sysconfdir')
```

Meson got fancy new path concatenation syntax in v0.50 :)